client/my-sites/plugins-wpcom/standard-plugins-panel.jsx

```diff
@@ -27,7 +31,8 @@ export const StandardPluginsPanel = React.createClass( {
 				<CompactCard className="wpcom-plugins__standard-panel">
 					<div className="wpcom-plugins__list">
 						{ shownPlugins.map( ( { name, descriptionLink, icon, category, description } ) =>
-							<StandardPlugin
+							<Plugin
+								onClick={ () => onClick( name ) }
```

Comment:
If [we add this](https://github.com/Automattic/wp-calypso/pull/4995/files/70ba8c8eca897730bbdc82700b0771359c226790#r61158480), then we can avoid the fat arrow here:
```javascript
onClick={ onClick }
```